mslib/mscolab/file_manager.py

```diff
@@ -250,6 +254,42 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def save_user_profile_image(self, user_id, image_file):
```

Comment:
this is now a dup of existing code in the chat_manager 
https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/chat_manager.py#L100

can you move this to the file_manager, refactor it so it can be used for all cases?